README.md

```diff
@@ -295,6 +352,7 @@ Built-in function ready to be bound to any key you like :smile:.
 | `builtin.treesitter`                | Lists Function names, variables, from Treesitter!                |
 | `builtin.live_grep`                 | Searches in current directory files (respecting .gitignore)      | 
 | `builtin.current_buffer_fuzzy_find` | Searches in current buffer lines.                                |
+| `builtin.current_buffer_tags`                      ||
```

Comment:
Tags in the current buffer

Reply:
:smile: you are awesome @Conni2461, I really appreciate you taking in depth review of the changes. :heart: 
For these description, I left them after doing some testing with the awesome stuff you've added. I'll mark reviews on desc as resolved as I'm about to push new commit